### PR TITLE
Http extension

### PR DESCRIPTION
**Pull request by nicolasbotto**

### CODE REVIEW

File: src/edge/modules/httpExtension/Controllers/ScoreController.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+using httpExtension.Processors;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Drawing;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace httpExtension.Controllers
```
Comment (ethanshea-ms) [inline src/edge/modules/httpExtension/Controllers/ScoreController.cs:12]: Namespace should be capitalized to follow C# convention

File: src/edge/modules/httpExtension/Readme.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+Once you've added the new class, you'll have to update the ScoreController so it instantiates your class and invokes the **ProcessImage** method on it to run your processing logic.
+
+### Building, publishing and running the Docker container
+
+To build the image, use the Docker file named `Dockerfile`.
+
+First, a couple assumptions
+
+* We'll be using Azure Container Registry (ACR) to publish our image before distributing it
+* Our local Docker container image is already loged into ACR.
```
Comment (ethanshea-ms) [inline src/edge/modules/httpExtension/Readme.md:49]: Should this read "Our local docker context is logged into our ACR"?

File: src/edge/modules/httpExtension/Readme.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        "name": "inferencingUrl",
+        "type": "String",
+        "description": "inferencing Url",
+        "default": "https://<REPLACE-WITH-IP-OR-CONTAINER-NAME>/score"
+      },
+```
+* Configuration
+```
+{
+	"@apiVersion": "1.0",
```
Comment (ethanshea-ms) [inline src/edge/modules/httpExtension/Readme.md:89]: nit: Indentation

I think this mixes tabs and spaces

File: src/edge/modules/httpExtension/Readme.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+          "scale":
+          {
+            "mode": "Pad",
+            "width": "416",
+            "height": "416"
+          },
+          "format":
+          {
+            "@type": "#Microsoft.Media.MediaGraphImageFormatEncoded",
+            "encoding": "jpeg",
```
Comment (ethanshea-ms) [inline src/edge/modules/httpExtension/Readme.md:115]: Using this, the images will be transferred with the JPEG encoding, but in the description at the top of the file, it says that decoded images are transferred.

File: src/edge/modules/httpExtension/Readme.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+}
+```
+
+First, a couple assumptions
+
+* We'll be using Azure Container Registry (ACR) to publish our image before distributing it
+* Our local Docker is already loged into ACR.
+* Our hypothetical ACR name is "myregistry". Your may defer, so please update it properly along the following commands.
+
+> If you're unfamiliar with ACR or have any questions, please follow this [demo on building and pushing an image into ACR](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-docker-cli).
```
Comment (ethanshea-ms) [inline src/edge/modules/httpExtension/Readme.md:131]: These lines are duplicated, is this intentional?

**Review by ethanshea-ms — CHANGES_REQUESTED** [review]
Make sure to incorporate comments in shared files from the below PR.

https://github.com/Azure-Samples/live-video-analytics-iot-edge-csharp/pull/26#pullrequestreview-517225701

**Comment by nicolasbotto** [comment]
@ethanshea-ms I've addressed your comments, please review them. Thanks

**Comment by naiteeks** [comment]
@ethanshea-ms  - good to merge?

**Comment by naiteeks** [comment]
@gadamilan  - for review.

**Pull request merged**